src/plugin.cpp

```diff
@@ -132,7 +132,7 @@ void RobocupPlugin::initialize(ed::InitData& init)
             ed::UpdateRequest req;
             std::stringstream error;
 
-            if (!model_loader_.create(id, type, req, error))
+            if (!model_loader_.create(id, type, req, error, true))
```

Comment:
What does `true` mean here?

Reply:
that it also looks for SDF models. Not only yamls

Reply:
Plz put that in a comment or if possible like a keyword-arg in python: `use_sdf=true`

Reply:
It is documented in the library. No need to put it here.